test/metabase/lib/test_metadata.cljc

```diff
@@ -117,24 +119,16 @@
                       :body       804      ; :type/Text
                       :product-id 805))))) ; :type/Integer
 
-(defmulti table-metadata
```

Comment:
Debugging `table-metadata` was annoying because it included all the `:fields`, so I changed this to a private `table-metadata-method` multimethod and added a separate `table-metadata` function that calls the multimethod and strips out irrelevant keys. As a bonus it now has Malli schema checking